apps/api/src/app/utils/getOAuthAccessToken.ts

```diff
@@ -0,0 +1,33 @@
+import { request } from "@bandada/utils"
+import { OAuthProvider } from "../reputation/types"
+
+/**
+ * It returns the access token for a given OAuth provider.
+ * @param oAuthProvider OAuth provider.
+ * @param oAuthCode OAuth code to exchange.
+ * @returns The access token.
+ */
+export default async function getOAuthAccessToken(
```

Comment:
Does it make sense to group these by the provider? That is one file github.ts containing the function for getting token and another for getting the accountId? The provider files could also be a class that implement a well defined interface, so other devs could just implement that to add new providers (without disturbing current files)

Maybe something to consider when we have more providers.

Reply:
Yes, that makes sense. I'll do it when I integrate Twitter as a second provider.